move-vm-backend/src/lib.rs

```diff
@@ -89,10 +92,30 @@ where
             anyhow!("Error code:{:?}: msg: '{}'", code, msg.unwrap_or_default())
         })?;
 
-        for (account, _, operation) in changeset.modules() {
+        for (account, identifier, operation) in changeset.modules() {
```

Comment:
let's leave a _TODO_ comment that this will be handled from the Warehouse

Reply:
Left 😄